libr/asm/arch/arm/armass16_const.h

```diff
@@ -0,0 +1,20 @@
+#define NONE 0x0
```

Comment:
from my experience this will break some builds, better use a prefix like AARM_ to every of those defines

Reply:
Ah, or maybe rather THUMB_. Will do...